invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/input.overrides

```diff
@@ -5,3 +5,45 @@
 .input.invitation-searchbar {
     width: 100%;
 }
+
+#header-search-bar {
+    .ui.fluid.search {
+        transition:all 0.3s;
+        width: @searchWidth;
+
+        &.active.visible, &.focus{
+            @media screen and (min-width: @largestTabletScreen) {
+                width: @activeSearchWidth;
+            }
+        }
+
+        .ui.input{
+            display: flex;
```

Comment:
It's not possible to apply directly to the component because the `search` cmp in react is wrapping the `input` div inside the `search` input, this tag is not directly declared as a React cmp.